### PR TITLE
Change `blaze` to `bazel` in the conflict error message

### PR DESCRIPTION
**Pull request by greggdonovan**

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by greggdonovan** [comment]
I signed it!

**Comment by mhlopko** [comment]
Hi Greg, thanks for the PR! Would you mind removing bazel from the error message completely? So it reads '...  run 'clean' and ...'? This way the error message makes sense also internally (where bazel is called blaze). Thanks!

File: src/main/java/com/google/devtools/build/lib/actions/ArtifactPrefixConflictException.java [hunk 1]
```diff
@@ -27,7 +27,7 @@ public ArtifactPrefixConflictException(PathFragment firstPath, PathFragment seco
     super(String.format(
         "output path '%s' (belonging to %s) is a prefix of output path '%s' (belonging to %s). "
         + "These actions cannot be simultaneously present; please rename one of the output files "
-        + "or, as a last resort, run 'blaze clean' and then build just one of them",
+        + "or, as a last resort, run 'bazel clean' and then build just one of them",
```
Comment (mhlopko) [inline src/main/java/com/google/devtools/build/lib/actions/ArtifactPrefixConflictException.java]: Remove bazel please

**Pull request closed without merging**